test/sync_basic_test.js

```diff
@@ -9,7 +9,7 @@ const ethTestUtils = require("./ethereum/utils");
 const utils = require("../src/common/utils");
 const crypto = require("../src/common/cryptography");
 
-const B2Path = path.join(__dirname, "./testUtils/id-l");
+const B2Path = path.join(__dirname, "./testUtils/id-l.json");
```

Comment:
Also, remove the weird `./` and split `"./testUtils/id-l.json"` to `"testUtils", "id-l.json"`.